.gitignore

```diff
@@ -9,3 +9,4 @@ dist/
 *.swp
 *~
 Pipfile
+.coverage
```

Comment:
I think it's better to make this in another patch.

Reply:
see #20